daffodil-test/src/test/scala/org/apache/daffodil/section13/packed/TestPacked.scala

```diff
@@ -100,4 +100,7 @@ class TestPacked {
   @Test def testDelimitedIBM4690DecSeqUnparser(): Unit = {
     runner.runOneTest("DelimitedIBM4690DecSeqUnparser")
   }
+
+  // Daffodil-2961
+  @Test def testBCDBigDecimalError(): Unit = { runner.runOneTest("bcdBigDecimalError") }
```

Comment:
Can you add some out of range tests as well? That's a new limitation that this will enforce that I think wasn't previously enforced.